Lib/multiprocessing/pool.py

```diff
@@ -255,7 +262,8 @@ def apply(self, func, args=(), kwds={}):
         '''
         Equivalent of `func(*args, **kwds)`.
         '''
-        assert self._state == RUN
+        if self._state != RUN:
```

Comment:
`apply_sync()` seems to already have this check, so we can probably remove it here.